app_core.py

```diff
@@ -120,6 +120,11 @@ def default(self, o):
 else:
     app.config["CSS_THEME_INTENSITY"] = "light"
 
+if os.getenv("MIN_AVAILABLE_NZD_BALANCE_CENTS"):
+    app.config["MIN_AVAILABLE_NZD_BALANCE_CENTS"] = os.getenv("MIN_AVAILABLE_NZD_BALANCE_CENTS")
```

Comment:
this will assign the config value as a string (which will crash the code that expects a numeric type)

```suggestion
    app.config["MIN_AVAILABLE_NZD_BALANCE"] = decimal.Decimal(os.getenv("MIN_AVAILABLE_NZD_BALANCE"))
```

Reply:
this has been changed to the above